maquetado/setup/scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
+}
```

Comment:
Ops! You had two lines after the `{`, you should have only deleted one and left one. You should leave an single empty line at the end of each file